recipe/build.sh

```diff
@@ -13,7 +13,7 @@ export EUPS_PATH="${PREFIX}/share/eups"
 
```

Comment:
We don't need to change the build script IIUIC. I don't think the changes here will hurt though.

Reply:
These were to pass shellcheck.  They help with things like pathnames that contain spaces.